apollo-ios/Sources/Apollo/CacheWriteInterceptor.swift

```diff
@@ -7,17 +7,12 @@ import ApolloAPI
 public struct CacheWriteInterceptor: ApolloInterceptor {
 
   public enum CacheWriteError: Error, LocalizedError {
-    @available(*, deprecated, message: "Will be removed in a future version.")
```

Comment:
This was deprecated because in the defer execution PR it was no longer being used. Had we caught this bug then it would never have been deprecated.